test/screenshotter/ss_data.js

```diff
@@ -0,0 +1,14 @@
+var fs = require("fs");
+var querystring = require("querystring");
+var jsyaml = require("js-yaml");
+
+var dict = fs.readFileSync(require.resolve("./ss_data.yaml"));
+dict = jsyaml.safeLoad(dict);
+for (var key in dict) {
+    var itm = dict[key];
+    if (typeof itm === "string") {
+        itm = dict[key] = { m: itm };
```

Comment:
Could we maybe rename this `m` parameter to `tex` or something? Now that it's easier to see what's going on, we might as well name things reasonably. :)